easybuild/easyconfigs/g/genometools/genometools-1.5.8-foss-2015b-no-cairo.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'MakeCp'
+
+name = 'genometools'
+version = '1.5.8'
+versionsuffix = '-no-cairo'
```

Comment:
In that case, let's just put this PR on hold until your problems with Python are resolved, and then get back to this.